announcements/_posts/2018-01-15-pyhep2018.md

```diff
@@ -6,4 +6,4 @@ symbol: glyphicon-calendar
 link: http://hepsoftwarefoundation.org/activities/pyhep.html
 until: 2018-07-08
 ---
-PyHEP 2018 "Python in HEP" Workshop, 7-8 July 2018, Sofia (Bulgaria)
+[PyHEP 2018 "Python in HEP" Workshop, 7-8 July 2018, Sofia (Bulgaria)](http://hepsoftwarefoundation.org/activities/pyhep.html)
```

Comment:
For consistency with the rest we could link to the indico page https://indico.cern.ch/event/694818/ instead. The link you give should become the general link for all PyHEP activities (reminds me the pages needs an update to have info on the Gitter channel for example. It's on my to-do list.)